ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/ccd/services/CoreCaseDataService.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.hmcts.cmc.ccd.migration.ccd.services;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.ccd.migration.ccd.services.exceptions.CreateCaseException;
+import uk.gov.hmcts.cmc.ccd.migration.ccd.services.exceptions.OverwriteCaseException;
+import uk.gov.hmcts.cmc.ccd.migration.idam.models.User;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.EventRequestData;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Service
+@ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
+public class CoreCaseDataService {
+
+    enum EventType {
+        MIGRATED_FROM_CLAIMSTORE("MigrationFromClaimstoreEvent");
```

Comment:
is the `Event` suffix needed in the value? I see its a bit hit and miss in the spreadsheet atm

Reply:
There are two approaches in the spreadsheet